### PR TITLE
Change EWRAM_DATA_2 to EWRAM_INIT

### PR DESCRIPTION
**Pull request by DizzyEggg**
I recently realized PMD-red has a cool thing, where it lets you assign EWRAM values other than 0(they're placed in rom during compile-time and then copied to ram at start-up). I changed linker script to keep that functionality.

### CODE REVIEW

**Review by AnonymousRandomPerson — APPROVED** [review]

**Comment by SethBarberee** [comment]
Needs rebase with recent merge.

**Comment by AnonymousRandomPerson** [comment]
https://github.com/pret/pmd-red/pull/286 just merged, which created some conflicts.

File: src/code_2.c [hunk 1]
```diff
@@ -38,10 +38,31 @@ extern void LoadTitleScreen(void);
...
+
+// TODO: figure out the struct
+struct UnkStruct203B040
+{
+    u8 unk0[48];
+};
+
+EWRAM_INIT OpenedFile *gTitlePaletteFile = NULL;
+EWRAM_INIT u32 gUnknown_203B03C = 0;
+EWRAM_INIT struct UnkStruct203B040 gPersonalityRelated_203B040 = {
```
Comment (SethBarberee) [inline src/code_2.c:52]: You can use the `PersonalityRelated` struct defined in `personality_test1.h` after the recent merge.

**Comment by DizzyEggg** [comment]
> #286 just merged, which created some conflicts.

Conflicts solved

**Pull request merged**